lib/index.js

```diff
@@ -13,5 +13,28 @@ module.exports = (dirname, manifestExtras) => {
 
     const manifest = Manifest.concat(manifestExtras || []);
 
-    return Haute(dirname, manifest);
+    return (server, options, next) => {
+
+        let reply;
+
+        if (typeof next !== 'undefined') {
+            reply = Haute(dirname, manifest)(server, options, next);
```

Comment:
Just `return` here and drop the `else {}`.  This will also allow you to get rid of the `reply` variable, which is a little bit confusingly named (since `reply` has a specific meaning in hapi-land).